frontend_app_workspace_advanced/src/component/SpaceDescription.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  AutoComplete,
+  IconButton,
+  tinymceRemove
+} from 'tracim_frontend_lib'
+import { translate } from 'react-i18next'
+
+export const SpaceDescription = (props) => {
+  const [showEditionMode, setShowEditionMode] = useState(false)
+
+  useEffect(() => {
+    if (showEditionMode) {
+      globalThis.wysiwyg(
+        `#${props.textareaId}`,
+        props.lang,
+        props.onChangeDescription,
+        props.onTinyMceInput,
+        props.onTinyMceKeyDown,
+        props.onTinyMceKeyUp,
+        props.onTinyMceSelectionChange
+      )
+    }
+  }, [props.lang, showEditionMode])
```

Comment:
if `lang` change, shouldn't `tinymceRemove` be called before calling `globalThis.wysiwyg`?

Reply:
Fixed